metadata.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "example42-java",
+  "version": "2.1.0",
+  "summary": "Puppet module to manage Java",
+  "license": "Apache-2.0",
+  "author": "Alessandro Franceschi, Martin Alfke",
+  "checksums": {
+  },
+  "source": "https://github.com/example42/puppet-java",
+  "project_page": "https://github.com/example42/puppet-java",
+  "issues_url": "https://github.com/example42/puppet-java/issues",
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Centos",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "14.04"
+      ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "10",
+        "11"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "pe",
+      "version_requirement": ">= 3.0.0 < 5.0.0"
+    },
+    {
+      "name": "puppet",
+      "version_requirement": ">= 3.0.0 < 5.0.0"
+    }
+  ],
+  "dependencies": [
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 3.2.0 < 5.0.0"
+    },
+    {
+      "name": "example42/puppi",
+      "version_requirement": ">= 2.0.0"
+    },
+    {
+      "name": "example42/monitor",
+      "version_requirement": ">= 2.0.0"
+    },
+    {
+      "name": "example42/iptables",
+      "version_requirement": ">= 2.0.0"
+    },
+    {
+      "name": "example42/firewall",
```

Comment:
Yes, actually the script used for pushing modules to the forge https://github.com/example42/Example42-tools/blob/master/publish.sh did that already. But it's better to remove them everywhere, or better, leave them commented, specifying they are optional.